src/modules/DataCatalog/components/List/Table/container.js

```diff
@@ -44,7 +44,7 @@ function mapStateToProps(state, ownProps) {
 }
 
 const mapDispatchToProps = {
-  showDatasource: datasource => goToPage(paths.dataCatalog(datasource.uuid)),
+  showDatasource: datasource => goToPage(paths.dataCatalog(datasource.centralId)),
```

Comment:
AT LEAST, there are more usages of DS uuid in Data Catalog and CDM Source List modules